features/admin.actions.v1/components/pre-update-password-action-config-form.tsx

```diff
@@ -119,6 +134,10 @@ const PreUpdatePasswordActionConfigForm: FunctionComponent<PreUpdatePasswordActi
             setAuthenticationType(initialValues.authenticationType as AuthenticationType);
             setIsAuthenticationUpdateFormState(false);
         }
+
+        if (initialValues?.rule) {
+            setIsHasRule(true);
```

Comment:
Can't we set this as the initial value of `rule` state variable and get rid of the `isHasRule`?

Reply:
@jeradrutnam WDYT on this suggestion ?